docs/input.md

```diff
@@ -59,4 +59,15 @@ Single libraries are assumed to have high quality and reasonable coverage. For e
 
 Note, that you should not specify PacBio CLR, Sanger reads or additional contigs as single-read libraries, each of them has a separate [option](running.md#input-data).
 
-See [examples](running.md#examples).
\ No newline at end of file
+See [examples](running.md#examples).
+
+## Reading NCBI SRA files
+
+NCBI SRA uses a binary format to store reads. SPAdes has an ability to read these files directly, so there is no need to convert them into intermediate FASTQ files. However, this brings additional dependency on NCBI SDK and therefore there might be compatibility issues. Overall, the following is required:
+
+  - SPAdes should be compiled with NCBI SDK support enabled (pass `-DSPADES_USE_NCBISDK=ON` to `spades_compile.sh`, disabled by default, enabled in release binaries).
+  - Input file should have `.sra` extension.
+  - Input file should be passed as interlaced (in case of paired-end reads) or single-end library.
+  - Read error correction does not support reading SRA files (so one should use `--only-assembler` command line option).
+
+SRA files could contain reads that were aligned to a reference. Reading of such files without a reference could be slow and SPAdes will produce a warning. For such files it is advised to  convert to FASTQ first.
```

Comment:
SRA files can contain reads that were aligned to a reference genome. Reading such files without a reference can be slow and SPAdes will produce a warning in this case. We recommend to convert such files to FASTQ first.